packages/terra-application-layout/src/terra-dev-site/doc/common/UtilityOption.scss

```diff
@@ -0,0 +1,7 @@
+@import './ApplicationLayoutCommon';
```

Comment:
We can name this file as ApplicationLayoutDocCommon.scss .

Reply:
Changed the file name to "ApplicationLayoutDocCommon.scss"